tests/__galvanic_test/jcli_wrapper.rs

```diff
@@ -0,0 +1,64 @@
+use std::process::Command;
+use std::path::PathBuf;
+
+use crate::__galvanic_test::configuration;
```

Comment:
What with the double underscore name?

Reply:
I still don't like the name. What's the purpose of the underscore?

Besides, this use statement could just use `super::` if I'm not mistaken.

Reply:
It's cleaner to use the `crate::path::to::feature`. It gives a real indication where it is in the crate. Simpler to find out where the data came from.